tests/integration/bond_test.py

```diff
@@ -36,6 +36,9 @@
 from .testlib import statelib
 from .testlib.assertlib import assert_mac_address
 
+from .linux_bridge_test import _linux_bridge
```

Comment:
In python we use a `_` prefix to declare it as private.
You cannot import private entities.